libr/core/cmd_anal.c

```diff
@@ -871,6 +871,7 @@ static bool cmd_anal_aaft(RCore *core) {
 	r_reg_arena_push (core->anal->reg);
 	// Iterating Reverse so that we get function in top-bottom call order
 	r_list_foreach_prev (core->anal->fcns, it, fcn) {
+		r_reg_arena_zero (core->anal->reg);
```

Comment:
i think it would be better to do that after `aei`

Reply:
What is the difference?

Reply:
by reading the code doesnt seems like there's anything in aeim that modifies the registers. so its fine to call .zero before aei and aeim. but we can speedup the whole thing by moving the aei/aeim outside, and just push/pop the zeroed state inside the loop. because we want to keep SP,BP registers